docstore/memdocstore/mem.go

```diff
@@ -140,9 +136,9 @@ func (c *collection) ErrorCode(err error) gcerr.ErrorCode {
 func (c *collection) RunActions(ctx context.Context, actions []*driver.Action, opts *driver.RunActionsOptions) driver.ActionListError {
 	errs := make([]error, len(actions))
 
-	// Run the actions concurrently with each other.
+	// Run the actions concurrently with each other, with a max of 10.
 	run := func(as []*driver.Action) {
-		t := driver.NewThrottle(c.opts.MaxOutstandingActionRPCs)
+		t := driver.NewThrottle(10)
```

Comment:
If we want to remove the max, why do we still need to throttling here?

Reply:
So, should I just remove this entirely? I wasn't sure if doing so would result in unbounded goroutines.

Reply:
As per my comment in "Conversation," can we just rename the option to `MaxOutstandingActions`?

Reply:
Done.